nixos/modules/services/cluster/druid/default.nix

```
@@ -0,0 +1,221 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.druid;
+
+  druidServiceOption = serviceName: {
+    enable = mkEnableOption (mdDoc serviceName);
+    restartIfChanged = mkOption {
+      type = types.bool;
+      description = mdDoc ''
```

```suggestion
      description = ''
```
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557